sdk/java-sdk-spring/src/main/scala/kalix/javasdk/impl/reflection/Reflect.scala

```diff
@@ -50,4 +55,32 @@ object Reflect {
     }
 
   }
+
+  private implicit val stringArrayOrdering: Ordering[Array[String]] =
+    Ordering.fromLessThan(util.Arrays.compare[String](_, _) < 0)
+
+  implicit val methodOrdering: Ordering[Method] =
+    Ordering.by((m: Method) => (m.getName, m.getReturnType.getName, m.getParameterTypes.map(_.getName)))
+
+  def lookupComponentClientField(instance: Any): List[ComponentClientImpl] = {
```

Comment:
why is this a list of ComponentClients? I would expect only a single one or none.

Reply:
Me too, but we have not guarantee that how many component client a user will try to inject.